docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,224 @@ title: Architecture
 
 ### Audit Event Definition File
 
 In order to use Auditing in an application, the application's auditing events must be specified along with the parameters that are associated with each of the events. These events are specified in an audit event definition file. You can read more about the audit event definition file and its XML schema in the [Getting Started Guide](Getting-Started.md).
 
-### Vertica Database Schema and Tables
-
-Providing the Audit Management web service /applications endpoint with the example audit event definition XML will create schemas and tables in the `CAFAudit` database for the application.
-
-![Audit Management Application Events Table With Sample Application](images/AuditManagementApplicationEventsWithSampleAppVertica.png)
-
-The above figure's `ApplicationEvents` table, under the `AuditManagement` schema, contains a row for a registered SampleApp's audit event definition XML. The `applicationId` column contains the Application ID provided in the audit event definition XML and the `eventsXML` column contains the XML passed to the API. Audit Management web service uses `eventsXML` to create a tenant's application table with columns to match the audit event data types.
-
-Registering a tenant with the Audit Management web service /tenants endpoint creates an entry under the `AuditManagement` schema's `TenantApplications` for the tenant.
-
-![Audit Management Tenant Applications Table With Tenant ID](images/AuditManagementTenantApplicationsWithTenantApplication.png)
-
-The figure shows a row for the registered tenant; `tenantId` with an associated application's `applicationId`. This table keeps track of which tenants use which applications.
-
-Registering a new tenant creates a new schema under the `CAFAudit` database for the tenant called `account_<tenantId>`, where audit event data for the tenant's applications will be held. Audit Management also creates a `kafka_rej` table for holding the tenant's rejected audit events.
-
-![CAF Audit Account 1 Sample App Table Columns](images/account_1AuditSampleAppColumns.png)
-
-The figure shows an `account_1` schema with an `AuditSampleApp` table and the columns for audit event data for the application.
-
-![CAF Audit Account 1 Kafka Reject Table Columns](images/account_1RejectTable.png)
-
-The figure shows the `account_1` schema with a `kafka_rej` table and columns for rejected audit event data.
+### Elasticsearch Indices and Type Mappings
+
+On a tenant's first call of the Audit library, an index is created for the them. A tenant's Elasticsearch index, type and document's ID (`index/type/_id`) is seen as `audit_tenant_<tenantId>/cafAuditEvent/<applicationAuditEvent>`.
+
+    GET /audit_tenant_1/_mapping/cafAuditEvent
+    {
+      "audit_tenant_1": {
+        "mappings": {
+          "cafAuditEvent": {
+            "dynamic_templates": [
+              {
+                "CAFAuditString": {
+                  "match": "*_CAStr",
```

Comment:
Chris, can you double check with Dermot over these suffixes to make sure they are def what we all agreed? Also the short suffix needs changes to something else as Dermot did not like this. Agree with him on this.